PySDM/backends/impl_numba/methods/pair_methods.py

```diff
@@ -72,7 +72,7 @@ def max_pair(data_out, data_in, is_first_in_pair, idx):
     @numba.njit(**conf.JIT_FLAGS)
     def min_pair_body(data_out, data_in, is_first_in_pair, idx, length):
         data_out[:] = 0
-        for i in numba.prange(length):  # pylint: disable=not-an-iterable
+        for i in numba.prange(length - 1):  # pylint: disable=not-an-iterable
```

Comment:
CC @abulenok 
let's have a test showing that it is needed (as it shouldn't be needed if `is_first_in_pair` is OK!)